packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -18,6 +18,22 @@
     }
 }
 
+.dc-input .dc-input {
```

Comment:
I think we don't need parent class 
```suggestion
.dc-input {
```

Reply:
it is needed to avoid using `!important`. Parent node has `dc-input` class

Reply:
Can we give this field a `className`?